stl/inc/algorithm

```diff
@@ -3851,10 +3861,14 @@ _CONSTEXPR20 _OutIt replace_copy(_InIt _First, _InIt _Last, _OutIt _Dest, const
     const auto _ULast = _STD _Get_unwrapped(_Last);
     auto _UDest       = _STD _Get_unwrapped_n(_Dest, _STD _Idl_distance<_InIt>(_UFirst, _ULast));
     for (; _UFirst != _ULast; ++_UFirst, (void) ++_UDest) {
-        if (*_UFirst == _Oldval) {
-            *_UDest = _Newval;
+        if constexpr (_Can_vectorize_replace_copy<decltype(_UDest), _Iter_value_t<_InIt>, _Ty>) {
+            *_UDest = (*_UFirst == _Oldval) ? _Newval : *_UFirst;

```

Comment:
The parentheses here are unnecessary; I don't *strongly* object to them, but they don't appear in the other expressions, and this one isn't very complicated.